builds/misc/metrics-collector-images-publish.yaml

```diff
@@ -0,0 +1,60 @@
+name: $(version)
+trigger: none
+pr: none
+
+variables:
+  NugetSecurityAnalysisWarningLevel: warn
+  
+jobs:
+  - deployment: publishImages
+    displayName: Publish Images
+    pool:
+      vmImage: ubuntu-18.04
+    workspace:
+      clean: all
+    environment: 'Azure-IoT-Edge-Core Release Env'
+    strategy:
+      runOnce:
+        deploy:
+          steps:
+          - checkout: self
+            clean: true
+            fetchDepth: 100
+        
+          - task: Bash@3
+            displayName: Log into Registries
+            inputs:
+              targetType: Inline
+              script: |
+                docker login $(registry.address) --username $(registry.user) --password $(registry.password)
```

Comment:
@and-rewsmith I created a service connection that you can try. Probably better to refer to the service connection through a variable rather than hardcoding it in the YAML (I say that because I see a pattern in our codebase of hardcoding the service connection name for our daily builds registry in several places, I'd like to get away from that).

Reply:
@and-rewsmith I guess the service connection name is already an abstraction around the real registry information...creating a variable for the service connection name might be overkill. Although if we hard code it, that means that anyone else who wants to use our YAML definitions in their own environment will need a service connection with a specific name.

Reply:
@damonbarry  I was looking into this particular issue a while ago. I think there is a work item on my backlogs. This bad practice can be found in various pipelines. We should just get rid of them all at once (& reset the passwords).

Reply:
@damonbarry I need to make it a pipeline variable anyway so that we can run the release pipeline without pushing to edgerelease.

Reply:
Thinking about this more, the manifest script needing the registry password complicates things because I would need to change it in order to not need it for this build but require it for others.

I feel the cleaner approach is to check this PR in without the service connection. Then I can make a separate PR to remove all the docker logins and alter the manifest script to not even take a password. This approach is also more cherry-pickable to master, whereas if we do it here it will be messier due to the 1.1 scoped metrics collector.